.github/workflows/commit_check.yaml

```
@@ -3,65 +1,67 @@
-              inject-env-vars: true
-              DOPPLER_PROJECT: ${{ secrets.DOPPLER_PROJECT }}
-              DOPPLER_CONFIG: ${{ secrets.DOPPLER_CONFIG }}
-          - uses: syphar/restore-virtualenv@v1
-            id: cache-virtualenv
-
-          - run: |
-                exit 0 
+    needs: setup-virtualenv
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)